environment.yml

```diff
@@ -30,3 +30,5 @@ dependencies:
  - xarray
  - shapely
  - pydap>3.2.2|<3.2.2
+ - cxx-compiler
+ - sysroot_linux-64
```

Comment:
does this have any impact depending on the operating system?

Reply:
I'm don't believe so, but I'm not completely confident. These packages should be segregated to the RAiDER environment from my understanding and shouldn't be affected by the operating system. Perhaps you could confirm/correct this @Askaholic ?

Reply:
I am not familiar with conda, I'll have to look it up.